presto-hive/src/main/java/com/facebook/presto/hive/HivePageSourceProvider.java

```diff
@@ -614,7 +653,8 @@ private static boolean shouldSkipBucket(HiveTableLayoutHandle hiveLayout, HiveSp
         return hiveBucketFilter.map(filter -> !filter.getBucketsToKeep().contains(hiveSplit.getReadBucketNumber().getAsInt())).orElse(false);
     }
 
-    private static boolean shouldSkipPartition(TypeManager typeManager, HiveTableLayoutHandle hiveLayout, DateTimeZone hiveStorageTimeZone, HiveSplit hiveSplit, SplitContext splitContext)
+    private static boolean shouldSkipPartition(TypeManager typeManager, HiveTableLayoutHandle hiveLayout, DateTimeZone hiveStorageTimeZone, HiveSplit hiveSplit, SplitContext
```

Comment:
nit: this function signature and the one below
```suggestion
    private static boolean shouldSkipPartition(TypeManager typeManager, 
    HiveTableLayoutHandle hiveLayout, 
    DateTimeZone hiveStorageTimeZone, 
    HiveSplit hiveSplit, 
    SplitContext splitContext)
```